http_parser.c

```diff
@@ -766,10 +768,6 @@ size_t http_parser_execute (http_parser *parser,
             UPDATE_STATE(s_res_H);
             break;
 
-          case CR:
-          case LF:
-            break;
-
```

Comment:
Now that the switch has only one case (well, two if you count the default) can you turn it into an if statement?